src/response/records.ts

```diff
@@ -62,6 +78,12 @@ export interface IpAddress extends Insights {
    * A higher score indicates a higher risk.
    */
   risk: number;
+  /**
+   * An array containing risk reason objects identifying the reasons why the IP
+   * address received the associated risk. This will be an empty list if there
+   * are no reasons.
+   */
+  riskReasons?: IpRiskReasons[];
```

Comment:
Will it be an empty list? (I don't know). If so do we need the '?'?

Reply:
Woops, that documentation isn't true. The warnings return null so I did the same here.